COORDINATION.md

```diff
@@ -0,0 +1,103 @@
+# Project Coordination Guidelines
+
+Arguably, communication is our primary concern throughout the entire process of
+designing, implementing, documenting, testing, and disseminating our programs.
+But, without question, collaboration on software development calls for a lot of
+communication.
+
+This document records our shared (always evolving) understanding of the key ways
+we communicate in furtherance of our collaboration. Its intention is to ensure that we are aligned between ourselves in regard to how and why we use the tools we do.
+
+## Elements of our project coordination
+
+We currently use GitHub as the main platform for sharing the communications
+that let us coordinate our work. GitHub has a number of different features for
+enabling cooperative work, but it's not necessarily obvious what the scope or
+intended meaning of each feature is. The following table is used to align our
+shared understanding of how to use and interpret the main features:
+
+| Feature                  | Scope                                                   | Communicates...        |
+|--------------------------|---------------------------------------------------------|------------------------|
+| [issue][issues]          | a distinct problem or task                              | what to work on        |
+| [pull request][prs]      | (part of) a solution or fulfillment of a task           | the ongoing work       |
+| [milestone][milestones]  | a distinct feature or a closely related set of problems | what work is towards   |
+| [label][labels]          | a more or less vague category/topic/quality             | attributes of the work |
+| [project board][project] | concurrent human processes with a common dynamic        | how work is done       |
+| [discussion][discussion] | free ranging and open discussion                        | whatever               |
+
+(For more on projects vs. milestones, see
+https://stackoverflow.com/a/47542346/1187277)
+
+[issues]: https://github.com/informalsystems/apalache/issues
+[prs]: https://github.com/informalsystems/apalache/pulls
+[milestones]: https://github.com/informalsystems/apalache/milestones
+[labels]: https://github.com/informalsystems/apalache/issues/labels
+[project]: https://github.com/orgs/informalsystems/projects/30
```

Comment:
I assume this document is going to be public, and since the project board is only accessible with the right permission, it might be good to point this out somehow.